tensorflow/lite/arena_planner_test.cc

```diff
@@ -210,6 +210,19 @@ TEST_F(ArenaPlannerTest, EmptyGraph) {
   Execute(0, 10);
 }
 
+TEST_F(ArenaPlannerTest, DeallocationOfInputTensor) {
+  // This is a negative TC, which will try to make sure that no allocation for
+  // input tensors is done, hence making it call
+  // CalculateDeallocationOfInternalTensors with negative node_index, since
```

Comment:
"negative node_index. Since the previous check..."

Reply:
Sure, same is updated in the code.

Regards
Amit